desktop/cmp/input/Select.ts

```diff
@@ -236,6 +243,10 @@ class SelectInputModel extends HoistInputModel {
         return !!this.componentProps.enableMulti;
     }
 
+    get emptyValue(): any {
+        return this.componentProps.emptyValue ?? null;
```

Comment:
the only case where this *might* be a bit surprising is if you set emptyValue to `undefined`, but I think this is actually fine as is